services/rook-ceph-cluster/1.10.3/grafana-dashboards/kustomization.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: ${releaseNamespace}
+configMapGenerator:
+  - name: grafana-dashboards-rook-ceph
+    namespace: ${releaseNamespace}
+    files:
+      - rook-ceph.json
```

```suggestion
      - rook-ceph-cluster.json
```

can we rename the json file to `rook-ceph-cluster.json` just to be consistent?

---

thanks